src/plugins/charts/bar/BarGraphView.vue

```diff
@@ -257,7 +257,8 @@ export default {
 
       return {
         end,
-        start
+        start,
+        size: 1
       };
```

Comment:
I think we should add `strategy: 'latest'` here. `strategy` provides a hint to providers, that in this case can be used to query a "latest data" table instead of a parameter archive, which can be a significant performance saving.

Reply:
This is why I asked you the question yesterday. :) OK will change it.

Reply:
Sorry, I meant that we want both. `strategy=latest` and `size=1` are two subtly different things.

Reply:
> Sorry, I meant that we want both. `strategy=latest` and `size=1` are two subtly different things.

@shefalijoshi @akhenry does this mean both `size=1` and `strategy=latest` ?

Reply:
Yes, always. strategy="latest" is a hint to fetch it from a LAD store if available. Both Yamcs and AMPCS have a Latest Available Data table in-memory for rapid access, distinct from the telemetry archive which is comparatively slow to access. size=1 just means "send me one result".